core/src/KokkosExp_MDRangePolicy.hpp

```diff
@@ -196,43 +212,45 @@ struct MDRangePolicy : public Kokkos::Impl::PolicyTraits<Properties...> {
   KOKKOS_INLINE_FUNCTION const typename traits::execution_space& space() const {
     return m_space;
   }
-  template <typename LT, typename UT, typename TT = array_index_type>
-  MDRangePolicy(std::initializer_list<LT> const& lower,
-                std::initializer_list<UT> const& upper,
-                std::initializer_list<TT> const& tile = {})
-      : m_space() {
-    init(lower, upper, tile);
+
+  MDRangePolicy() = default;
+
+  template <typename LT, std::size_t LN, typename UT, std::size_t UN,
+            typename TT = array_index_type, std::size_t TN = rank>
```

Comment:
I am tempted to SFINAE on these being integral types but that would be an API change.  Any thought on this?

Reply:
I would be fine with that if we deprecate the interface for non-integral types.

Reply:
I think we can constrain directly to integral types. It is a bug if you give this non-integral types I think.